src/app/login/form.tsx

```diff
@@ -46,38 +44,47 @@ export const LoginForm = () => {
     }
   };
 
-  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
-    const { name, value } = event.target;
-    setFormValues({ ...formValues, [name]: value });
-  };
-
   return (
-    <Form onSubmit={onSubmit}>
-      <Stack>
-        <InputText
-          placeholder='Email'
-          name='email'
-          type='email'
-          onChange={handleChange}
-          value={formValues.email}
-          autoFocus={true}
-          required={true}
-        />
-        <InputText
-          placeholder='Password'
-          name='password'
-          type='password'
-          onChange={handleChange}
-          value={formValues.password}
-          autoFocus
-          required
-        />
-        {error && <p>{error}</p>}
-        <Button type='submit' disabled={loading} color='primary'>
-          {loading ? <Spinner /> : <FaSignInAlt />}
-          Sign In
-        </Button>
-      </Stack>
-    </Form>
+      <Box
+          sx={{
+            marginTop: 8,
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'center',
+          }}
+      >
+        <Typography component="h1" variant="h5">
+          Sign in
+        </Typography>
+        <Box component="form" onSubmit={onSubmit} noValidate sx={{ mt: 1 }}>
+          <TextField
+              margin="normal"
```

Comment:
Should these be single quotes? Consider autorun prettier on save or setting husky precommit